staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -344,8 +345,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ResilientWatchCacheInitialization: {Default: true, PreRelease: featuregate.Beta},
 
-	RetryGenerateName: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
shouldn't this also be added to the versioned feature gates struct above?

Reply:
https://github.com/kubernetes/kubernetes/blob/f38f106ab31dc6300bd88c8e04f2f1cbe9a5d136/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L306-L311